k8s/environments/prod/cluster-00-overlay/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+# - ../../../release/admin/kube-slack
+# - ../../../release/admin/traefik
+# - ../../../release/admin/kured
+- ../../../release/admin/keyvault-flexvol
```

Comment:
can we not deploy this please? we shouldn't be using it and best not to put it in prod by accident